src/git.md

```diff
@@ -238,8 +238,7 @@ rebase! Use `git push --force-with-lease` instead.
 
 ## Advanced Rebasing
 
-If your branch contains multiple consecutive rewrites of the same code, or if
-the rebase conflicts are extremely severe, you can use
+If your branch conflicts are extremely severe, you can use
```

Comment:
I think this was good advice, we shouldn't remove it. It's not *normative*, but I know several reviewers that will ask you to clean up the history if it changes the same thing in place in different ways.